specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.0/examples/SuccessfulSentimentRequest.json

```diff
@@ -0,0 +1,127 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "Endpoint": "{Endpoint}",
+    "input": {
+      "documents": [
+        {
+          "language": "en",
+          "id": "1",
+          "text": "Hello world. This is some input text that I love."
+        },
+        {
+          "language": "en",
+          "id": "2",
+          "text": "It's incredibly sunny outside! I'm so happy."
+        },
+        {
+          "language": "en",
+          "id": "3",
+          "text": "Pike place market is my favorite Seattle attraction."
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "documents": [
+          {
+            "id": "1",
+            "sentiment": "positive",
+            "documentScores": {
```

Comment:
@ctufts to verify, but I believe this has also changed to confidenceScores.
Both `documentScores` and `sentenceScores`.

Reply:
Please ignore, just saw Chris's comment below ;)

Reply:
Updated